src/main/java/eu/dissco/backend/repository/ElasticSearchRepository.java

```diff
@@ -79,15 +86,15 @@ public List<AnnotationResponse> getLatestAnnotations(int pageNumber, int pageSiz
     var offset = getOffset(pageNumber, pageSize);
     var pageSizePlusOne = pageSize + ONE_TO_CHECK_NEXT;
     var searchRequest = new SearchRequest.Builder()
-        .index(ANNOTATION_INDEX)
+        .index(properties.getAnnotationIndex())
         .sort(s -> s.field(f -> f.field(FIELD_CREATED).order(SortOrder.Desc))).from(offset)
         .size(pageSizePlusOne).build();
     return client.search(searchRequest, ObjectNode.class).hits().hits().stream().map(Hit::source)
         .map(this::mapToAnnotationResponse).toList();
   }
 
   private AnnotationResponse mapToAnnotationResponse(ObjectNode json) {
-    var annotation = json.get(ANNOTATION_INDEX);
+    var annotation = json.get("annotation");
```

Comment:
No this is not an index call. We 'misused' that static variable because everything was called "anntation" but not this is the only place the string is there